### ISSUE TITLE
Gripper stuck when grasping object

### ISSUE THREAD

**Issue by JonathanYang0127** [facebookresearch/fairo#1398]
## Type of Issue

Select the type of issue:
- [x] Bug report (to report a bug)
- [x] Feature request (to request an additional feature)
- [ ] Tracker (I am just using this as a tracker)
- [ ] Refactor request
- [ ] Documentation Ask

## Description

When using the gripper goto function (regardless of blocking), the franka hand gripper gets stuck when asked to fully close when there is an object between its fingers. As a result, it does not respond to all future commands because it continuously attempts to reach the fully closed position.

## Expected Behavior

With blocking off, the gripper should respond to a new command regardless of whether the old one was achieved.

**Comment by AlexanderKhazatsky** [comment]
I'm experiencing this issue as well. Specifically, I'm experiencing two issues:

1) The gripper pauses unnecessarily when going to a new width (ex from fully open to fully closed)
2) When the gripper tries to grasp an object, if it isn't able to fully able to go to the desired location, it gets stuck - even after I update the desired state to be fully open. It only starts to move after I remove the object blocking the gripper fingers from closing.

**Comment by AlexanderKhazatsky** [comment]
Here’s some videos of the issues discussed:


https://user-images.githubusercontent.com/30299984/201435469-aa006e6f-0fdd-457d-9e85-42ada36fbd30.MOV

**Comment by AlexanderKhazatsky** [comment]
https://user-images.githubusercontent.com/30299984/201436229-9497d19c-6e17-46a0-bcf7-3f93b7e2ba3f.mov

**Comment by AlexanderKhazatsky** [comment]
Wanted to bump this issue :)

**Comment by exhaustin** [comment]
Hello sorry for the slow response. I cannot change the behavior of `goto` for the Franka Hand as we are simply wrapping around [franka::Gripper::move](http://docs.ros.org/en/kinetic/api/libfranka/html/classfranka_1_1Gripper.html#a331720c9e26f23a5fa3de1e171b1a684).
What I can do is to expose [`franka::Gripper::grasp`](http://docs.ros.org/en/kinetic/api/libfranka/html/classfranka_1_1Gripper.html#a19b711cc7eb4cb560d1c52f0864fdc0d), which is designed to avoid this issue when grasping. I've done that in the PR mentioned above and will close the issue since there is nothing I can do more than that.

So basically, the thing with the Franka Hand is that you can either move to a specific width (and get blocked until it errors out if the gripper is unable to reach the width), or close the gripper all the way until the fingers hit something.

Thanks as always for the detailed description of the issue, I hope your fingers are fine :)
